docs/integrate-cardano/creating-wallet-faucet.md

```diff
@@ -175,14 +175,14 @@ Now that we have a **wallet address**, we can then query the **UTXO** of the add
 
 ```bash
 cardano-cli query utxo \
---mary-era \
+--shelley-mode \
```

Comment:
I'd think it's best to remove `--shelley-mode` as well from doco, as the defaults are now elected correctly by CLI

Reply:
Thanks for the tip. Just pushed a new commit.